integration/js/app/video_test/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This should be at least under the app folder. I dont think we need to create a new package for every test app.

Reply:
Done.